test/integration/publication_test.rb

```diff
@@ -21,14 +19,17 @@ class PublicationTest < ActionDispatch::IntegrationTest
   test "renders metadata and document footer" do
     setup_and_visit_content_item('publication')
 
-    assert_has_component_metadata_pair("first_published", "3 May 2016")
-    link1 = "<a href=\"/government/organisations/environment-agency\">Environment Agency</a>"
-    link2 = "<a href=\"/government/people/eric-pickles\">The Rt Hon Sir Eric Pickles MP</a>"
-    assert_has_component_metadata_pair("from", [link1, link2])
-    assert_has_component_document_footer_pair("from", [link1, link2])
+    within(".app-c-publisher-metadata") do
+      assert page.has_css?(".app-c-published-dates", text: "Published 3 May 2016")
+
```

Comment:
might be worth checking both links that were originally tested to ensure we can still/continue to render more than one when necessary

Reply:
@maxgds done here now: https://github.com/alphagov/government-frontend/pull/558/commits/11ac2f352ff32e1f769aea1ee90d97c205d38384#diff-af4ea7d9ecb0773ef795162d376a087bR27